MapboxSearch.podspec

```diff
@@ -24,5 +24,5 @@ Some iOS platform specifics applies.
 
```

Comment:
pls update `s.ios.deployment_target` to 12.0 in both podspecs

Reply:
Oh that should have been pulled in from SSDK-499. I think I need to adjust my git configuration for the next merge.